source/MRViewer/ImGuiMenu.cpp

```diff
@@ -2280,11 +2295,11 @@ void ImGuiMenu::make_width( std::vector<std::shared_ptr<VisualObject>> selectedV
     }
     const auto valueConstForComparation = value;
 
-    ImGui::PushItemWidth( 40 * menu_scaling() );
+    ImGui::PushItemWidth( 50 * menu_scaling() );
     if ( lineWidth )
         UI::drag<PixelSizeUnit>( label, value );
     else
-        UI::drag<PixelSizeUnit>( label, value, 0.02f, 1.0f, 10.0f );
+        UI::drag<PixelSizeUnit>( label, value, 0.02f, 1.0f, 25.0f );
```

Comment:
please update master, there will be conflict here

Reply:
fixed